websites/F/feds/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "1105810827224809532",
+		"name": "for#4726"
+	},
+	"service": "feds",
+	"description": {
+		"en": "feds.lol is a freemium biolink service."
+	},
+	"url": "feds.lol",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/nBWzgJs.png",
```

Comment:
Logos [must be 512x512](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo)